src/main/java/view/LottoOutputView.java

```diff
@@ -18,19 +15,26 @@ public static void printLottoList(List<Lotto> lottoList) {
         lottoList.forEach(System.out::println);
     }
 
-    public static void printLottoResult(List<LottoResult> results) {
-        System.out.println("당첨 통계");
-        System.out.println("---------");
+    public static void printLottoResult(LottoStatistics statistics) {
+        StringBuilder builder = new StringBuilder();
+        builder.append("당첨 통계\n");
+        builder.append("---------\n");
 
-        Arrays.stream(LottoResult.values())
-                .filter(lottoResultType -> lottoResultType != LottoResult.NONE)
-                .sorted(Comparator.comparingInt(LottoResult::matchCount))
-                .forEach(lottoResultType -> {
-                    System.out.println(lottoResultType.matchCount() + "개 일치 (" + lottoResultType.winningMoney() + "원) - " + results.stream()
-                            .filter(result -> result == lottoResultType)
-                            .count() + "개");
-                });
+        final Map<LottoResult, Long> lottoResultMap = statistics.groupAndCounting();
 
-        System.out.printf("총 수익률은 %.2f입니다.%n", LottoManager.calculateProfitRate(LottoSeller.LOTTO_PRICE * results.size(), LottoManager.sumOfLottoResult(results)));
+        LottoResult.findByWinning()
+                .forEach(e -> builder.append(createLottoResult(e, Optional.ofNullable(lottoResultMap.get(e)).orElse(0L))));
+
```

Comment:
```suggestion
        LottoResult.findByWinning()
                .forEach(e -> builder.append(createLottoResult(e, lottoResultMap.getOrDefault(e, 0L))));
```

기본으로 제공하고 있는 메서드가 있으니 사용해보세요~